front_end/css/styles.css

```diff
@@ -2,7 +2,7 @@
     font-family: 'Lato', sans-serif;
 }
 
-.ace_editor p, .ace_editor div { 
+.ace_editor * {
```

Comment:
The issue with the offset ace editor cursor I found was due to CSS styles overriding our font-family: monospace requirement. This edit solves the problem and doesn't seem to cause any unintentional changes from my checking a few pages.